front/src/components/UI/ProductPrices/ProductPrices.tsx

```diff
@@ -0,0 +1,83 @@
+import { FieldsetStyledWrapper, Information, StyledInput, Col1, Col2 } from "./ProductPrices.styled"
+
+const ProductPrices = () => {
+  return (
+    <FieldsetStyledWrapper>
+      <Col1>
```

Comment:
Tutaj dobrze widać że można to rozbić na 2 komponenty. Są nawet zgrupowane w Col1 i Col2 (+te nazwy również wymagają poprawy, gdy w nazwie komponentu pojawia się cyfra, to znaczy że coś semantycznie jest nie tak).